### ISSUE TITLE
Can't subset columns that don't exist

### ISSUE THREAD

**Issue by luca-dex** [caravagn/mobster#30]
In `plot_latent_variables` a MOBSTER FIT is passed as main argument and it contains a tibble like

```
# A tibble: 3 x 7
     VAF cluster   Tail       C1       C2 is_driver driver_label
   <dbl> <chr>    <dbl>    <dbl>    <dbl> <lgl>     <chr>       
1 0.448  C1      0.0125 9.88e- 1 8.08e-21 TRUE      DR1         
2 0.159  C2      0.225  2.35e-34 7.75e- 1 TRUE      DR2         
3 0.0629 Tail    1.00   1.91e-82 4.02e- 5 TRUE      DR3 
```

after passing the objet to the function `Clusters` we get

```
# A tibble: 5,000 x 10
     VAF cluster Tail...3 C1...4   C2...5 is_driver driver_label Tail...8 C1...9
   <dbl> <chr>      <dbl>  <dbl>    <dbl> <lgl>     <chr>           <dbl>  <dbl>
 1 0.497 C1       0.00736  0.993 5.22e-27 FALSE     NA            0.00736  0.993
 2 0.490 C1       0.00669  0.993 4.42e-26 FALSE     NA            0.00669  0.993
 3 0.470 C1       0.00705  0.993 1.31e-23 FALSE     NA            0.00705  0.993
 4 0.517 C1       0.0130   0.987 1.83e-29 FALSE     NA            0.0130   0.987
 5 0.506 C1       0.00903  0.991 3.86e-28 FALSE     NA            0.00903  0.991
 6 0.440 C1       0.0179   0.982 9.68e-20 FALSE     NA            0.0179   0.982
 7 0.428 C1       0.0347   0.965 3.88e-18 FALSE     NA            0.0347   0.965
 8 0.523 C1       0.0164   0.984 3.97e-30 FALSE     NA            0.0164   0.984
 9 0.482 C1       0.00648  0.994 3.87e-25 FALSE     NA            0.00648  0.994
10 0.499 C1       0.00759  0.992 3.20e-27 FALSE     NA            0.00759  0.992
# … with 4,990 more rows, and 1 more variable: C2...10 <dbl>
```

This results in the break of the execution of 

```
clusters_names = names(x$pi)
assignments %>% select(clusters_names)
```

where `assignments` is the results of the `Cluster` function.
The two tibbles have different column names and the select doesn't work causing an error.

The problem can be replicated executing the second stage of the vignette "2. Plotting fits"

**Comment by caravagn** [comment]
I am sorry but cannot replicate the error. I am running

```
data('fit_example', package = 'mobster')

mobster::Clusters(fit_example$best)
mobster::plot_latent_variables(fit_example$best, cutoff_assignment = 0)
```
which works. Can you tell me where you get the crash?

**Comment by luca-dex** [comment]
Try rexecuting the code of `a2_plotting.Rmd` with the installed version of mobster, not with `document()`.

This is how the `best_fit` is generated:

```R
example_data = Clusters(mobster::fit_example$best)

# Drivers annotation (we selected this entries to have nice plots)
drivers_rows = c(2239, 3246, 3800)

example_data$is_driver = FALSE
example_data$driver_label = NA

example_data$is_driver[drivers_rows] = TRUE
example_data$driver_label[drivers_rows] = c("DR1", "DR2", "DR3")

# Fit and print the data
fit = mobster_fit(example_data, auto_setup = 'FAST')

best_fit = fit$best
```
And then:

```R
mobster::plot_latent_variables(best_fit, cutoff_assignment = 0)
```

You can see the error on all the environment in the actions

**Comment by caravagn** [comment]
I am not sure @luca-dex. I have done a fresh clean install of the development repo, restarted my R session, and run this smoothly. 

```
data("fit_example", package = 'mobster')
example_data = mobster::Clusters(mobster::fit_example$best)

drivers_rows = c(2239, 3246, 3800)

example_data$is_driver = FALSE
example_data$driver_label = NA

example_data$is_driver[drivers_rows] = TRUE
example_data$driver_label[drivers_rows] = c("DR1", "DR2", "DR3")

# Fit and print the data
fit = mobster::mobster_fit(example_data, auto_setup = 'FAST')

best_fit = fit$best
mobster::plot_latent_variables(best_fit, cutoff_assignment = 0)
```

**Comment by luca-dex** [comment]
Fixed with 378c6f0
